README.md

```diff
@@ -1,32 +1,56 @@
-# `@ubiquity/ts-template`
+# `@ubiquibot/command-start-stop`
 
-This template repository includes support for the following:
+This plugin allows a hunter to begin a task as well as gracefully stop a task without incurring a negative impact on the hunter's XP or karma.
 
-- TypeScript
-- Environment Variables
-- Conventional Commits
-- Automatic deployment to Cloudflare Pages
+## Usage
 
-## Testing
+### Start a task
 
-### Cypress
+To start a task, a hunter should use the `/start` command. This will assign them to the issue so long as the following is true:
 
-To test with Cypress Studio UI, run
+- Price labels are set on the issue
+- The issue is not already assigned
+- The hunter has not reached the maximum number of concurrent tasks
+- The command is not disabled at the repository or organization level
+- TODO: If the hunter meets the required XP requirements
 
-```shell
-yarn cy:open
-```
+### Stop a task
+
+To stop a task, a hunter should use the `/stop` command. This will unassign them from the issue so long as the following is true:
+
+- The hunter is assigned to the issue
+- The command is not disabled at the repository or organization level
+- The command is called on the issue, not the associated pull request
 
-Otherwise to simply run the tests through the console, run
+### [Configuration](./src/types/plugin-input.ts)
 
-```shell
-yarn cy:run
+#### Note: The command name is `"start"` when configuring your `.ubiquibot-config.yml` file.
+
+To configure your Ubiquibot to run this plugin, add the following to the `.ubiquibot-config.yml` file in your organization configuration repository.
+
+```yml
+- plugin: http://localhost:4000 # or the URL where the plugin is hosted
+  name: start-stop
+  id: start-stop-command
+  description: "Allows a user to start/stop a bounty without negative XP impact"
```

Comment:
```suggestion
  description: "Allows a user to start/stop a task without negative XP impact"
```

We completely stopped using "bounty" or "bounties" because this is a crypto-native term and our aspiration is to work with non crypto developers as soon as possible.